frictionless/steps/field/field_update.py

```diff
@@ -50,11 +50,16 @@ class field_update(Step):
     A descriptor for the field to set the metadata.
     """
 
+    pass_row: bool = False
+    """
+    Pass the entire row as a parameter for the transformation function when True.
```

Comment:
I understand from the test that the row parameter is passed _in addition_ to the value, which is not obvious in this description.

I suggest to remove this ambiguity.

```suggestion
    Pass the entire row as a second parameter for the transformation function if set to True.
    The first parameter (the value) remains unchanged.
```

Reply:
Thank you for the suggestion, it is a valid change and I have committed it.